src/components/CollaborationBar/Collaborators/Collaborators.tsx

```diff
@@ -36,21 +37,37 @@ import {
   ToolbarAvatarGroup,
   ToolbarAvatarMoreButton,
 } from '../../common/Toolbar';
+import { ToolbarAvatarPresenter } from '../../common/Toolbar/ToolbarAvatarPresenter';
 import { ellipsis } from '../../common/ellipsis';
 import { orderMembersByState } from './orderMembersByState';
 
 export function Collaborators() {
   const { t } = useTranslation();
+  const { state } = usePresentationMode();
   const ownUserId = useWidgetApi().widgetParameters.userId;
+  const isViewingPresentation = state.type === 'presenting' || 'presentation';
+  const presentingUser =
+    state.type === 'presentation'
+      ? state.presenterUserId
+      : state.type === 'presenting'
+        ? ownUserId
+        : undefined;
```

Comment:
I would suggest to rename this variable to `presenterUserId` since it holds id of the user and not the user entity